packages/atlas/src/views/viewer/VideoView/VideoView.styles.ts

```diff
@@ -1,26 +1,53 @@
 import styled from '@emotion/styled'
 import { Link } from 'react-router-dom'
 
+import { GridItem } from '@/components/LayoutGrid'
+import { LayoutGrid } from '@/components/LayoutGrid'
 import { LimitedWidthContainer } from '@/components/LimitedWidthContainer'
 import { Text } from '@/components/Text'
 import { Button } from '@/components/_buttons/Button'
 import { CallToActionWrapper } from '@/components/_buttons/CallToActionButton'
 import { SkeletonLoader } from '@/components/_loaders/SkeletonLoader'
 import { cVar, media, sizes } from '@/styles'
 
-export const PlayerContainer = styled.div`
+type CinematicView = {
+  cinematicView: boolean
+}
+
+export const PlyerGridWrapper = styled(LimitedWidthContainer)<CinematicView>`
+  max-width: ${({ cinematicView }) => cinematicView && 'unset'};
+  padding-bottom: 0;
+  margin-bottom: ${({ cinematicView }) => cinematicView && sizes(8)};
+`
+
+export const PlayerGridItem = styled(GridItem)`
   width: 100%;
-  height: calc(100vw * 0.5625);
+`
+
+export const PlayerWrapper = styled(LayoutGrid)<CinematicView>`
+  display: ${({ cinematicView }) => !cinematicView && 'flex'};
+  justify-content: ${({ cinematicView }) => !cinematicView && 'center'};
+  margin: 0 ${({ cinematicView }) => (cinematicView ? 'calc(-1 * var(--size-global-horizontal-padding))' : 0)};
+
   ${media.md} {
-    height: calc((100vw - var(--size-sidenav-width-collapsed)) * 0.5625);
-    max-height: calc(70vh);
+    padding-top: ${({ cinematicView }) => !cinematicView && sizes(8)};
+  }
+
+  ${media.xl} {
+    padding-top: ${({ cinematicView }) => !cinematicView && sizes(16)};
```

Comment:
Same here, let's move all conditional styles to a single block to avoid all of those conditions